css/src/components/layout.scss

```diff
@@ -4,6 +4,15 @@ $quarter-widescreen: math.div($breakpoint-widescreen, 4);
 $half-widescreen: math.div($breakpoint-widescreen, 2);
 $three-quarters-widescreen: math.div($breakpoint-widescreen, 4) * 3;
 
+.site-header {
```

Comment:
remove this

Reply:
Nah, it's probably fine to keep around! 😜

Reply:
Removed!